templates/webapi/main/index.html.ep

```diff
@@ -83,20 +83,11 @@
 
             <div class="form-group">
```

Comment:
It would make sense to add the `form-inline` class here so the input box has a reduced width and the "seconds" is not wrapped.